index.js

```diff
@@ -10,7 +10,7 @@
 */
 var path = require('path'),
     extend = require('deep-extend'),
-    osHomedir = require('os-homedir'),
+    osHomedir = require('os').homedir(),
```

Comment:
hmm... this requires deprecation of support for ancient versions of Node that didn't have `os.homedir()`. It still might be the right thing to do.

Reply:
https://www.npmjs.com/package/os-homedir is deprecated and `os.homedir()` was introduced in the io.js days. We can make this change.